src/components/ContentSidebar/Skills/SidebarSkills.js

```diff
@@ -25,50 +20,20 @@ type Props = {
     onInteraction: Function
 };
 
-function getCard(
-    skill: SkillCard,
-    getPreviewer: Function,
-    rootElement: HTMLElement,
-    appElement: HTMLElement,
-    onInteraction: Function
-) {
-    const { skill_card_type, error } = skill;
-    const onSkillInteraction = (data: any) => {
-        onInteraction({
-            skill: cloneDeep(skill),
-            interaction: cloneDeep(data)
-        });
-    };
+const skillError = {
+    errorHeader: <FormattedMessage {...messages.skillUnknownError} />
+};
 
+/**
+ * Parses a skil error into a MaskError
+ * @param {*} error
+ */
+const getError = (error?: any) => {
     if (error) {
-        return (
-            <span className='be-skills-error'>
-                <FormattedMessage {...messages.skillUnknownError} />
-            </span>
-        );
+        return skillError;
     }
-
-    switch (skill_card_type) {
-        case 'keyword':
-            return <Keywords skill={skill} getPreviewer={getPreviewer} onInteraction={onSkillInteraction} />;
-        case 'keyvalue':
-            return <Keyvalues skill={skill} />;
-        case 'timeline':
-            return <Timelines skill={skill} getPreviewer={getPreviewer} onInteraction={onSkillInteraction} />;
-        case 'transcript':
-            return (
-                <Transcript
-                    skill={skill}
-                    getPreviewer={getPreviewer}
-                    rootElement={rootElement}
-                    appElement={appElement}
-                    onInteraction={onSkillInteraction}
-                />
-            );
-        default:
-            return null;
-    }
-}
+    return undefined;
```

Comment:
Can possibly be simplified and embedded inline below

to maskError={card.error || skillError || null}